dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -185,6 +185,8 @@ public static ConnectorFactory admin() {
 
     private String bindHost = null;
 
+    private boolean inheritChannel = false;
```

Comment:
Could you add a description of this option in `HttpConnectorFactory` javadoc? It would be consistent with the rest of options, which are documented.